mitodl/mit-open
# Copies static assets to the root build directory

### Description (What does it do?)


Copies frontend static assets (currently just robots.txt and favicon.ico) to the root build directory during the frontend build. Previously these went into the ./static directory. Nginx on Heroku had a rule to check this directory for requests for files at root, though not the case with Fastly + S3.



### How can this be tested?


- Deploy to RC
- Confirm that https://mitopen-rc.odl.mit.edu/robots.txt loads
- Confirm that https://mitopen-rc.odl.mit.edu/favicon.ico loads
